anaconda_lib/helpers.py

```diff
@@ -208,7 +208,7 @@ def get_settings(view, name, default=None):
     if (name in ('python_interpreter', 'extra_paths') and not
             ENVIRON_HOOK_INVALID[view.id()]):
         if view.window() is not None and view.window().folders():
```

Comment:
Not quite sure if this should be changed as well. `view.window()` is used later, but I'm not savvy w/ sublime plugins.